src/applications/facility-locator/components/MessagePromptDiv.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+const MessagePromptDiv = ({ id, message, waitBeforeShow = 0 }) => {
+  const [isShown, setIsShown] = useState(false);
+
+  useEffect(
+    () => {
+      setTimeout(() => {
+        setIsShown(true);
+      }, waitBeforeShow);
+    },
+    [waitBeforeShow],
+  );
+
+  return isShown ? (
+    <div className="dropdown" role="alert" id={id}>
+      <div className="vads-u-margin--1p5">{message}</div>
+    </div>
+  ) : null;
+};
+
+MessagePromptDiv.propTypes = {
+  id: PropTypes.string,
+  message: PropTypes.string,
```

Comment:
Should `waitBeforeShow` get added to `propTypes`?

Reply:
Yes, I can add it. 
Thanks @Rleahy22 !